eagle-server/pom.xml

```diff
@@ -129,6 +129,14 @@
                     <groupId>org.wso2.orbit.com.lmax</groupId>
                     <artifactId>disruptor</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>slf4j-log4j12</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>slf4j-simple</artifactId>
+                </exclusion>
```

Comment:
We should not include `slf4j-simple` for most cases. And dropwizard forces using logback, so the slf4j may not work at all.